caliban/docker.py

```diff
@@ -391,9 +391,17 @@ def _notebook_entries(lab: bool = False, version: Optional[str] = None) -> str:
 
   library = "jupyterlab" if lab else "jupyter"
 
-  return """
+  if not dlvm:
+    return """
 RUN pip install {}{}
```

Comment:
oh, wait, good catch that we need up date this `pip` too!  I can do that separately.

Reply:
We have this `--lab` flag. I'm thinking that we should only do this special installation if `--lab` is specified. If not, even with a `dlvm` base image, we should just install normal `jupyter`. wdyt?